eng/WpfArcadeSdk/tools/ApiCompat.targets

```diff
@@ -0,0 +1,55 @@
+<Project>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.ApiCompat" Version="$(MicrosoftDotNetApiCompatVersion)"/>
```

Comment:
Can we condition this `PackageReference` on `"'$(RunNetFrameworkApiCompat)' == 'true'"` ?

Reply:
Sure, makes sense.